docs/release_notes/0.2.0.md

```diff
@@ -0,0 +1,85 @@
+in August 2018, and we are switching to a new release cadence now.
+Albeit we have originally planned to do a number of things in 0.2.0,
+the plan has evolved. We will start publishing short-term roadmaps
+in due course for each of planned minor releases.
+
+Please also note that this release has no new features since 0.1.39.
+
+## Acknowledgment
+
+Weaveworks team would like to sincerely thank everyone who has
+contributed to the project, your work is highly appreciated and
+the project would not have gotten this far without you!
+
+We have received many very useful contributions from so many of
+you, some of which came as code, and some as issues or participation
+on Slack, we thank all of you!
+
+Most active contributors in the last 12 month were:
+
+@richardcase @christopherhein @mumoshu @adamjohnson01 @archisgore @Lazyshot
+@kschumy @toricls @cristian-radu @tedmiston @karinnainiguez @Chabane @tiffanyfay
+@yutachaos @gruebel @jstrachan @marccarre @PaulMaddox @ozzieba @mukaibot @jmcarp
+@gotjosh @philoserf @pawelprazak @nckturner @af12066 @mgalgs @Jeffwan @paulbsch
+@arbourd @cbluth @knorby @gchaincl @mcfedr @overdrive3000 @prageethw @patstrom
+@sdarwin @unguiculus @Yannig @IPyandy @bowlesns @derwasp @mikeroyal @superseb
+@zironycho @dcherman @silviogutierrez @sixth @dresnick-sf @jonk @danielfm
+@denwwer @thapakazi @lucioveloso @callmeradical @manabusakai @justincormack
+@austbot @pdavies011010 @mreferre @flou @polothy @Tyil @cpaika @arun-gupta
+@cdenneen @danielchalef @mrichman @whereisaaron @StevenACoffman @procyclinsur
+@braderhart @aparamon @JasonSwindle @jicowan @olipachi @trondhindenes @milkowski
+@dingn1 @jamesalbert @stevepe-1 @ArseniiPetrovich @jvanzyl @redborian @tkang007
+@elirankon @alexclifford @arielvinas @bnutt @voxxit 
+
+## New Features Since 0.1.0
+
+### General features
+
+- create and manage clusters using configuration files
+- Kubernetes versions: 1.11, 1.12 and 1.13
+- cluster upgrades
+- support for all EKS regions
+
+### VPC
+
+- allow re-using existing VPCs
+- public and private subnet topologies
+- NAT Gateway support (HA, single or disable)
+
+### nodegroup
+
+- nodegroup management (create, get, scale, delete)
+- spot instances and mixed instance types
+- extra kubelet configuration parameters
+- custom EC2 tags and Kubernetes labels
+- custom bootstrap logic
+- single-AZ nodegroups
+- public and private networking options
+- use existing security groups
+- selecting root EBS volume size, type and encryption
+- various options for SSH access
+
+### IAM
+
+- use custom service role for cluster
+- use custom roles or policies for nodegroups
+- canned policies for popular add-ons:
+  - External DNS
+  - Cluster Autoscaler
+  - Cert Manager
+  - ALB Ingress
+  - AppMesh & XRay
+  - EBS, EFS and xFS CSI
+- execute CloudFormation actions using dedicated role
+
+### utilities
+
+- describe and troubleshoot CloudFormation stacks, with CloudTrail support
```

Comment:
I didn't realise we did this, is in in `eksctl` or is it that I can go to CloudTrail and see the logs from the CF stack?

Reply:
Yeah we have this, it's quite nice, but the feature is unpolished. We have an issue to improve it, it just needs nicer output format.

Reply:
Try `eksctl utils describe-stacks --trail`. It also supports `--event`.

Additionally, we do print events when things go wrong (which could use a little more polishing too (see #491).